### DISCUSSION TITLE
add -remove argument for alert commands

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#7717] [Enhancement]
currently we got the command
```
cmi alert (playerName) (Reason)
```

we are using an external plugin to do timed things and execute commands when the time is over

so to not nees to remove alerts each time from the chat interface it would be nice so have a remove arguments that removes the alert by command.

so i suggest
```
cmi alert (playerName) (Reason) (-remove) (-s)
```

permission ```cmi.commamd.alert.remove```

when the -remove Argument is used the alert (or alert of the player if defined) should be removed.

and about the -s

well i think i dont need to explain what it should be able to do. so many command allready have it. so it would be a nice extra for this command =)

**Comment by MrCrazys** [comment]
> currently we got the command
> 
> ```
> cmi alert (playerName) (Reason)
> ```
> 
> we are using an external plugin to do timed things and execute commands when the time is over
> 
> so to not nees to remove alerts each time from the chat interface it would be nice so have a remove arguments that removes the alert by command.
> 
> so i suggest
> 
> ```
> cmi alert (playerName) (Reason) (-remove) (-s)
> ```
> 
> permission `cmi.commamd.alert.remove`
> 
> when the -remove Argument is used the alert (or alert of the player if defined) should be removed.
> 
> and about the -s
> 
> well i think i dont need to explain what it should be able to do. so many command allready have it. so it would be a nice extra for this command =)

Already exists. This isn't necessary.

**Reply by CloudeLecaw** [reply]
what did i missed something? a remove Argument is not existing or nowhere listed and a -s too.

This is why its called enhancement of an existing feature  and not a new feature requests

![Screenshot_2023-03-30-16-31-05-310_com android chrome-edit](https://user-images.githubusercontent.com/12080017/228870129-f9e05fb0-dcfa-4852-852e-1e1a3f719632.jpg)

At least this is a request suggestion. so to tell its not nessesarry if you cant show how the "allready existing" looks like is no fair tread from you. at least -remove and -s do not exist for this command.
suggestion are there to be made. everyone can make them :)

**Reply by MrCrazys** [reply]
> what did i missed something? a remove Argument is not existing or nowhere listed and a -s too.
> 
> This is why its called enhancement of an existing feature and not a new feature requests
> 
> ![Screenshot_2023-03-30-16-31-05-310_com android chrome-edit](https://user-images.githubusercontent.com/12080017/228870129-f9e05fb0-dcfa-4852-852e-1e1a3f719632.jpg)
> 
> At least this is a request suggestion. so to tell its not nessesarry if you cant show how the "allready existing" looks like is no fair tread from you. at least -remove and -s do not exist for this command. suggestion are there to be made. everyone can make them :)

It is possible...

**Reply by CloudeLecaw** [reply]
> > what did i missed something? a remove Argument is not existing or nowhere listed and a -s too.
> > This is why its called enhancement of an existing feature and not a new feature requests
> > ![Screenshot_2023-03-30-16-31-05-310_com android chrome-edit](https://user-images.githubusercontent.com/12080017/228870129-f9e05fb0-dcfa-4852-852e-1e1a3f719632.jpg)
> > At least this is a request suggestion. so to tell its not nessesarry if you cant show how the "allready existing" looks like is no fair tread from you. at least -remove and -s do not exist for this command. suggestion are there to be made. everyone can make them :)
> 
> It is possible...

it doent matter. if its removed by recall.   i asked for a command Options because it would be needed by me.
not because something is possible in other was or not.

**Reply by MrCrazys** [reply]
> > > what did i missed something? a remove Argument is not existing or nowhere listed and a -s too.
> > > This is why its called enhancement of an existing feature and not a new feature requests
> > > ![Screenshot_2023-03-30-16-31-05-310_com android chrome-edit](https://user-images.githubusercontent.com/12080017/228870129-f9e05fb0-dcfa-4852-852e-1e1a3f719632.jpg)
> > > At least this is a request suggestion. so to tell its not nessesarry if you cant show how the "allready existing" looks like is no fair tread from you. at least -remove and -s do not exist for this command. suggestion are there to be made. everyone can make them :)
> > 
> > 
> > It is possible...
> 
> it doent matter. if its removed by recall. i asked for a command Options because it would be needed by me. not because something is possible in other was or not.

Why implement something that's already possible? That makes zero sense to me.

**Reply by CloudeLecaw** [reply]
> > > > what did i missed something? a remove Argument is not existing or nowhere listed and a -s too.
> > > > This is why its called enhancement of an existing feature and not a new feature requests
> > > > ![Screenshot_2023-03-30-16-31-05-310_com android chrome-edit](https://user-images.githubusercontent.com/12080017/228870129-f9e05fb0-dcfa-4852-852e-1e1a3f719632.jpg)
> > > > At least this is a request suggestion. so to tell its not nessesarry if you cant show how the "allready existing" looks like is no fair tread from you. at least -remove and -s do not exist for this command. suggestion are there to be made. everyone can make them :)
> > > 
> > > 
> > > It is possible...
> > 
> > 
> > it doent matter. if its removed by recall. i asked for a command Options because it would be needed by me. not because something is possible in other was or not.
> 
> Why implement something that's already possible? That makes zero sense to me.

it dont need to make sense for you. i need it as extra command argument. so i request it. Its not up to you to make an request bad like you do it here.
Everyone can make requests if he feel or needs something. and its allways up to Zrips to decide this.

**Reply by MrCrazys** [reply]
It's already a thing, as I have said multiple times. It has already been added. And I am not making this request "bad", I'm simply stating that what you are requesting is already a feature. What you're trying to say makes no sense 😛

**Comment by CloudeLecaw** [comment]
so where is the remove arument in the command?
as long
as a -remove does not exists your telling nonsense.
so try using -remove or show me the patchlog where it tells -remove was added

--------------------------------------------------------------------------

**Edit:**
I closed this request and reopend a new discussion request..
Thanks to this user this discussion has nothing to do anymore with a request. 
He made it more like a blame of a request.

A request is not made to blame others with it "this is nonsense or not nessesary or anything else"
If you feel like this, than simply dont answer and keep it for yourself in your brain.

I said it aggain to you. A Request is made because someone wants something and not because someone else feel like its not needed.
And its not up to you to downward other ideas with "this is nonsense, etc" just because may you had a bad day
or may because your requests got denied or never answered, or why every you feel like this.

**Reply by mrfloris** [reply]
Sorry to hear you two clashed and that you felt you had to close this to try and control the situation. I think it wasn't needed.

Please check the notable-role chat channel on Discord where I've poked you both.
